store/migration/dev/20220419000000__sheet_organizer.sql

```diff
@@ -0,0 +1,12 @@
+-- sheet_organizer table stores the sheet status data from a principal.
+CREATE TABLE sheet_organizer (
+    id SERIAL PRIMARY KEY,
+    sheet_id INTEGER NOT NULL REFERENCES sheet (id),
```

Comment:
Do we also need

```
    creator_id INTEGER NOT NULL REFERENCES principal (id),
    created_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
    updater_id INTEGER NOT NULL REFERENCES principal (id),
    updated_ts BIGINT NOT NULL DEFAULT extract(epoch from now()),
```

Reply:
I think we don't need `creator_id` or `updater_id`, because we already have `principal_id` which could only be the row's creator and updater. 
And we could need `create_ts` and `updated_ts`, let me add them later.

Reply:
But if we only have `updated_ts`, how can we sort the `starred` and `pinned`? Do we also need `starred_ts`/`pinned_ts`? wdyt

Reply:
> But if we only have `updated_ts`, how can we sort for `starred` and `pinned`? Do we also need `starred_ts`/`pinned_ts`? wdyt

Reply:
I don't feel we need those, similar to inbox table.